src/spirc.rs

```diff
@@ -172,6 +247,10 @@ impl SpircInternal {
                     self.is_active = false;
                     self.player.stop();
                 }
+
+                if frame.has_ident() && frame.has_goodbye() {
```

Comment:
I'm suprised this is a `kMessageTypeNotify`. Do `kMessageTypeGoodbye` messages ever happen ?